app/services/foreman/parameters/validator.rb

```diff
@@ -13,6 +13,10 @@ def validate!
           validate_regexp
         when "list", "array"
           validate_list
+        when "hash"
```

Comment:
I still see the "is invalid" error after entering your suggestion as the default value, and get the same logging as I pasted above.